docs/api.md

```diff
@@ -51,7 +48,40 @@ All the original static methods of the component are hoisted.
 
 ###### `getWrappedInstance(): ReactComponent`
 
-Returns the wrapped component instance. Only available if you pass `{ withRef: true }` as part of the `connect()`’s second `options` argument.
+Returns the wrapped component instance. Only available if you set `connect.defaults({ withRef: true })`.
+
+### `connect.defaults([newDefaults])`
+
+Returns a new `connect` which will have its defaults amended according to `newDefaults`.
+
+Calls to `.defaults()` can be chained, each latter call overriding defaults set in previous ones:
```

Comment:
Yep, I had trouble phrasing this. Not sure if "merging" necessarily implies the overwriting. I added the example for that reason. I'll change to merge anyway in both places to keep it consistent.